learn/configuration/instance_options.md

```diff
@@ -371,14 +371,18 @@ Sets the maximum size of [accepted payloads](/learn/core_concepts/documents.md#d
 
 ### Schedule snapshot creation
 
-::: warning
-🚩 This option does not take any values. Assigning a value will throw an error. 🚩
-:::
-
 **Environment variable**: `MEILI_SCHEDULE_SNAPSHOT`
 **CLI option**: `--schedule-snapshot`
+**Default value**: `false` or `86400` (24 hours)
+**Expected value**: a boolean or an integer
+
+Activates scheduled snapshots. Snapshots are disabled by default.
+
+It is possible to use `--schedule-snapshot` as a flag. If `--schedule-snapshot` is present when launching an instance but has not been assigned a value, Meilisearch takes a new snapshot every 24 hours.
 
-Activates scheduled snapshots when provided. Snapshots are disabled by default.
+It is also possible to explicitly pass a boolean value to `--schedule-snapshot`. Meilisearch takes a new snapshot every 24 hours when `--schedule-snapshot=true`, and takes no snapshots when `--schedule-snapshot=false`.
```

Comment:
Meilisearch will not accept `--schedule-snapshot=true` and `--schedule-snapshot=false`.

The possible alternatives are:
| Alternative | Effect |
|------------|-------|
| Do not pass `--schedule-snapshot` | No snapshot scheduled, unless specified otherwise in the configuration file or in the `MEILI_SCHEDULE_SNAPSHOT` environment variable |
| Pass `--schedule-snapshot` without a value | Snapshots scheduled with the default interval of about 24 hours |
| Pass `--schedule-snapshot x` with `x` an integer | Snapshots scheduled with the specified  `x` interval, in seconds |

The specification PR [does mention passing boolean values](https://github.com/meilisearch/specifications/pull/214/files#diff-e73478132c0f427ea0b612bbfaa4eb24f23801cb515d20c7daf340dd9f72656aR286), but this is only applicable to the `schedule_snapshot` variable in the configuration file (`config.toml`).

Hope this clears up things!